src/primitives/transaction.h

```diff
@@ -127,13 +127,72 @@ class CTxIn
     std::string ToString() const;
 };
 
+
+/**
+ * CAmount is an int64_t, which requires alignment of 8. This wrapper stores the data
+ * in a memory array and packs/unpacks it whenever needed, regarldess of alignment.
+ *
+ * Since CAmount's size is know, the compiler should be able to optimize the std::memcpy away
```

Comment:
Should be "known".
... And, uh, is that "away" or "anyway"?

Reply:
fixed in 03cb535384eb6d2f4284524c33ab03371bd263e2, also it should be "optimized away" because the compiler can the function call away. See this: https://godbolt.org/z/6uP9_S